miniapps/nurbs/nurbs_int.cpp

```diff
@@ -0,0 +1,164 @@
+//                                MFEM Example interpolation -- modified for NURBS FE
+//
+// Compile with: make nurbs_int
```

Comment:
This file is not compiled with `make` in the `nurbs` directory. I guess it needs to be added to `makefile`.

Reply:
app is removed